xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-pageobjects/src/main/java/org/xwiki/flamingo/sking/test/po/ExportModal.java

```diff
@@ -49,8 +51,15 @@ public ExportModal()
      */
     public OtherFormatPane openOtherFormatPane()
     {
-        getDriver().findElementByLinkText(OTHER_FORMAT_PANE_LINK).click();
-        getDriver().waitUntilElementIsVisible(By.cssSelector("#exportModelOtherCollapse .panel-body"));
+        WebElement linkPane = getDriver().findElementByLinkText(OTHER_FORMAT_PANE_LINK);
+        linkPane.click();
+
+        // sometimes it looks like the first click does not work.
```

Comment:
This looks like a hack. Would be nice to understand why the first click doesn't work. Maybe the link is moving (e.g. if there is some animation). AFAIK the click is done in 2 steps:

* it first gets the rectangular area that fits the button / link and computes the center point
* sends a click event in the center point

This kind of simulates the user behavior:

* you locate the button visually on the page and move the mouse over it
* you click

The issue is that the button / link may be moved between the steps, so you end up clicking on something else.

Reply:
It's absolutely a hack: I really don't understand why sometimes the click don't work the first time. Might be related to the fading of the modal (even though it should be managed in the modal PO). Might be the accordion, but don't see why.
It's the only solution I found to make the test passing 10 times on 10.